order_as_specified.gemspec

```diff
@@ -17,7 +17,6 @@ Gem::Specification.new do |spec|
 
   spec.files         = `git ls-files -z`.split("\x0")
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
```

Comment:
Fixes `order_as_specified.gemspec:20:3: C: [Correctable] Gemspec/DeprecatedAttributeAssignment: Do not set test_files in gemspec.`